src/release/breaking-changes/notifications.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Removing Notification.visitAncestor
+description: Notifications only traverse parents that are notification
```

Comment:
here and elsewhere: change parent to ancestor?

Reply:
(since it is explicitly no longer a parent walk but a jump over ancestors)

Reply:
Done